python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -938,6 +969,19 @@ def _get_unique_target_path(self, *, context: Optional[OpExecutionContext]) -> P
 
         return current_target_path.joinpath(path)
 
+    def auto_defer_args(self) -> Sequence[str]:
```

Comment:
Could we see this be called in `.cli(...)`, on behalf of the user? Probably need to add an argument to let the user toggle whether these arguments are added to their invocation.

Reply:
[1] the opt-in or this calling the method to get the args and adding them to your cli invocation, I got rid of the `auto` wording